designs/virtualization/4.x/expose-guest-data/expose guest data.md

```diff
@@ -0,0 +1,55 @@
+# Expose Guest Agent Data
+
+When the Guest Agent is installed on a VM some additional valuable data can be collected then surfaced in the UI.
+When the Guest Agent isn’t installed we will inform the users about that.
+
+We can report that in the VM’s Overview tab under the Status card and provide a message: “This virtual machine does not have a Guest Agent installed. Some metrics, and management features will not be available or inaccurate” and add an external link to view the details about the installation procedure of the Guest Agent, if the users are interested to do so. The link will lead the users to the instructions on how to install the Guest Agent.
+
+![Notifying users that Guest Agent isn't installed](img/vm-overview-tab.png)
+
+## After installing the Guest Agent
+
+Some of the data that is collected via the Guest Agent can be surfaced in the VM’s Overview tab under the Details card, as well as on the Details tab:
+
+- Hostname
+- Guest Agent OS version - the actual OS version that is installed on the VM
+- Time zone
+- Users list - active users that are currently logged in to the VM
+
+### VM’s Overview tab > Details card
+
+![Additional data when Guest Agent is installed](img/vm-dashboard-w-ga-installed.png)
+
+Clicking the number of logged in users will lead to a table with all the relevant info about these logged in users. This info is required so that users will be aware  when they're about to take an action that will affect those logged in users (f.e.) powering the VM off or restarting it.
+
+### Details tab
+
+![VM overview page](img/vm-overview.png)
+
+To simplify the information about OS versions (the Guest Agent and what was installed by the admin while creating the VM), we can surface only the current OS version that’s installed (the Guest Agent OS). 
+In case we have a mismatch between the chosen Operating System version (the OS that the admin originally installed) and the detected Operating System version, we can show an alert that indicates that mismatch and offer a solution.
+
+![notification when mismatch os version between Guest Agent and original](img/vm-overview-GA-not-match.png)
```

Comment:
In the image, the error message suggests we can identify a problem because of the missmatch ... but we can not.
My suggestion is to keep the warning text to what we do know, for example, "Operating System mismatch, identified X operating system installed on a virtual machine created for Y operating system."

Reply:
I agree, we should be only notifying them that the chosen OS does not match what the VM is actually running. Maybe make it a bit more readable though:
`Operating system mismatch`
`This virtual machine is running Windows but was configured to run Linux when it was created. `
WDYT @yaacov ?

Reply:
> WDYT @yaacov ?

@matthewcarleton sounds much better, thanks :+1:

Reply:
Thanks @yaacov & @matthewcarleton